File: eFormAPI/Plugins/Vehicles.Pn/Vehicles.Pn/Infrastructure/Models/VehicleInspectionPnModel.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Vehicles.Pn.Infrastructure.Models
{
    public class VehicleInspectionPnModel
    {

        public int Id { get; set; }
        public string WorkflowState { get; set; }
        public int Version { get; set; }
        public int? Status { get; set; }
        public DateTime? CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
        public int CreatedByUserID { get; set; }
        public int UpdatedByUserID { get; set; }
        public DateTime? DoneAt { get; set; }
        public int VehicleContractId { get; set; }
        public int SDKCaseId { get; set; }

    }
}

File: eform-client/src/app/plugins/modules/vehicles-pn/models/vehicleInspection-pn.model.ts
export class VehicleInspectionPnModel {
  id: number;
  workflowState: string;
  version: number;
  status: number;
  createdAt: Date;
  UpdatedAt: Date;
  DoneAt: Date;
  EformId: number;
  VehicleId: number;


  constructor() {
    this.createdAt = null;
    this.UpdatedAt = null;
    this.DoneAt = null;
  }
}
